frontend/src/pages/Posts.jsx

```
@@ -0,0 +1,326 @@
+          </div>
+        </div>
+
+        {/* Latest Job Openings */}
+        <div className="p-6 pt-0">
+          <h2 className="text-2xl font-semibold mb-4">
+            {loading ? "Loading job openings..." : "Latest Job Openings"}
+          </h2>
+
+          {loading ? (
+            <div className="flex justify-center items-center h-64">
+              <div className="animate-spin rounded-full h-12 w-12 border-t-2 border-b-2 border-blue-500"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)